modules/create-policy-categories.adoc

```diff
@@ -1,18 +1,42 @@
 // Module included in the following assemblies:
 //
 // * operating/manage-security-policies.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="create-policy-categories_{context}"]
 = Creating policy categories
 
 You can create new policy categories from the system policies view.
 
+{product-title-short} version 3.74 provides a new method to create and manage policy categories in {product-title-managed} or in {product-title-short} if you have the PostgreSQL database (Technical Preview) enabled. All policy workflows other than policy creation remain unchanged when using this feature. For instructions on using this feature, see the second procedure in the following section.
+
+You can also configure policy categories by using the `PolicyCategoryService` API object. For more information, navigate to *Help* -> *API reference* in the {product-title-short} portal. 
+
+:FeatureName: PostgreSQL support
+include::snippets/technology-preview.adoc[]
+
 .Procedure
-. On the RHACS portal, navigate to *Platform Configuration* -> *Policies*.
+
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Policy Management*.
 . From the *Policies* page, select the policy you want to edit.
 . Select *Actions* -> *Edit policy*.
-. Enter a new category name in the *Categories* field and then click *Create <category>*.
+. In the *Policy details* section, enter a new category name in the *Categories* field and then click *Create <category>*. 
++
+[NOTE]
+====
+This action is not available in {product-title-short} version 3.74 if you have the PostgreSQL database (Technical Preview) enabled. You must create new policy categories by using the procedure in the next section.
+====
 . Click the *Review policy* section heading.
 . Click *Save*.
 
-After you save the configuration, the new category displays on the dashboard if there are any violations for policies in the category.
+To create policy categories in {product-title-managed} or if you have the PostgreSQL database feature (Technical Preview) enabled:
```

Comment:
This is adding a second/third procedure to a single procedure module. This should be broken out into its own procedure module (or could be made into a single procedure by the workaround I mentioned above above).

Though now that I actually look at it closer, they do seem separate enough (creating vs editing) and different products or diff features enabled. I think they need to be broken up somehow better